src/inausoft.netCLI/Deserialization/RegexOptionsDeserializer.cs

```diff
@@ -34,23 +34,32 @@ public object Deserialize(Type type, string optionsExpression)
 
             if (!ValidateOptionsExpression(optionsExpression))
             {
-                throw new FormatException($"{nameof(optionsExpression)} has invalid format.");
+                throw new DeserializationException(ErrorCode.InvalidOptionsFormat, $"Cannot deserialize into type : {type} - specified option format : {OptionsPattern} is invalid.");
```

Comment:
Remove spaces before colon `:`. Perhaps colon itself could be also removed.

Reply:
Sure.